templates/base.html

```diff
@@ -282,6 +284,11 @@
                                     </ul>
                                 </div>
                             </li>
+                            <li>
+                                <a href="{% url "opinion:list" %}" class="mobile-menu-link {% block menu_opinion %}{% endblock %}">
+                                    {% trans "Tribunes libres" %}
```

Comment:
Pas faux. ça me va

Le ven. 13 mai 2016 14:46, SpaceFox <notifications@github.com> a écrit :

> In templates/base.html
> <https://github.com/zestedesavoir/zds-site/pull/3606#discussion_r63176318>
> :
>
> > @@ -282,6 +284,11 @@
> >                                      </ul>
> >                                  </div>
> >                              </li>
> > +                            <li>
> > +                                <a href="{% url "opinion:list" %}" class="mobile-menu-link {% block menu_opinion %}{% endblock %}">
> > +                                    {% trans "Tribunes libres" %}
>
> Ben là tu vois le contenu des tribunes des membres ; je me vois mal mettre
> au singulier et casser le sens parce qu'on arrive directement sur les
> contenus des tribunes, ni mettre une page intermédiaire pour des raisons de
> sémantiques.
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/zestedesavoir/zds-site/pull/3606/files/518dca2262e39978ea0e805f1fbe3a4f45c969ce#r63176318>
>